assignments/callbacks.js

```diff
@@ -23,25 +23,52 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 
 function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
+  // getLength passes the length of the array into the callback.\
+  cb(arr.length);
 }
 
+getLength(items, (lengthOfList) => {
+  console.log(lengthOfList);
+});
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  cb(arr[arr.length-1]);
 }
 
+last(items, (lastItem) => {
+  console.log(lastItem);
+});
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
-}
+    cb(x + y);
+    }
+  
+  sumNums(2, 6, sumOfXY => {
```

Comment:
this one is also missing the = sign, you could also get rid of the curly bracket and just put it on one line to keep it nice and short. Remember that in an arrow function, if you only have one expression to pass through then the 'return' is implied and the brackets aren't needed, (weird huh), so you don't have to add it in.